Resources/config/services.yml

```diff
@@ -3,6 +3,14 @@ services:
         class: AlexDpy\AclBundle\Manager\AclIdentifier
         arguments:
             - @security.context
+            - @security.acl.provider
```

Comment:
Create a single container parameter wich contains an array of all tables is a better way IMO, and that avoid to overload service definition files.